xgi/stats/__init__.py

```diff
@@ -117,6 +117,9 @@ def name(self):
     def __iter__(self):
         return iter(self._val.items())
 
+    def items(self):
+        return iter(self._val.items())
```

Comment:
Does this need to be wrapped in an iter?

Reply:
Good q - I think the answer is that it does not need to. But I chose to put it in there so it returns the same thing (specifically, the same type), as __iter__. 

...but now that I think back on it, it should probably better return the same type as `dict.items`, not the same type as `NodeStat.__iter__` :thinking:

Reply:
done.